docs/Nutrition1.md

```diff
@@ -0,0 +1,58 @@
+# Meal Planner Documentation
+
+## Overview
+
+## Step 1
```

Comment:
Why are we logging in here if the task is in the admin ui?

Reply:
This has been corrected